sdk/textanalytics/Azure.AI.TextAnalytics/src/Generated/Models/AbstractiveSummaryDocumentResult.Serialization.cs

```diff
@@ -36,7 +36,7 @@ void IUtf8JsonSerializable.Write(Utf8JsonWriter writer)
             if (Optional.IsDefined(Statistics))
             {
                 writer.WritePropertyName("statistics"u8);
-                writer.WriteObjectValue(Statistics.Value);
+                writer.WriteObjectValue(Statistics);
```

Comment:
What changed in autorest.csharp? Is there an overload specifically for `T?` or something? Otherwise, I'm concerned of the impact this would have.

Reply:
`WriteObjectValue` accepts `object` as a parameter, so it doesn't matter if we pass `T` or `T?` - it will be cast to `T` anyway. `Optional.IsDefined` checks `value.HasValue`, so there should be no NRE.

Reply:
I see, given the `IUtf8JsonSerializable` try-cast as demonstrated in https://gist.github.com/heaths/9bdb318ac14e311eed441e196bb1e9b2 (I had to check, and lifted our `WriteObjectValue` extension method with a few unused code paths elided).